screens/settings/NotificationsScreen.js

```diff
@@ -0,0 +1,171 @@
+import { FontAwesome5 } from '@expo/vector-icons';
+import AsyncStorage from '@react-native-community/async-storage';
+import { useFocusEffect } from '@react-navigation/native';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { ScrollView, View } from 'react-native';
+import AlertAsync from 'react-native-alert-async';
+import { Switch } from 'react-native-paper';
+import {
+  Body,
+  ButtonLabel,
+  FilledButtonContainer,
+  NavButtonContainer,
+  NavHeaderContainer,
+  NavTitle,
+  Subtitle,
+} from '../../components/BaseComponents';
+import Colors from '../../constants/Colors';
+import { getCustomersById, updateCustomers } from '../../lib/airtable/request';
+import { logErrorToSentry } from '../../lib/logUtils';
+import { ContentContainer, ResourceItemCard } from '../../styled/resources';
+
+export default function NotificationsScreen(props) {
+  const { navigation } = props;
+  const [notifications, setNotifications] = useState(false);
+  const [deliveryAlerts, setDeliveryAlerts] = React.useState(false);
+  const [notifsChanged, setNotifsChanged] = useState(false);
```

Comment:
I have these 'changed' variables to check if edits have been made so that the save button will pop up/disappear only if changes have been made, + it will show a warning if you try to exit without saving changes. Now that I think about it, that kind of feels like overkill.... so is this a bad thing to do?